src/Slider.js

```diff
@@ -9,14 +9,15 @@ import {
   Animated,
   StyleSheet,
   PanResponder,
-  View
+  View,
+  Easing
 } from "react-native";
 
-var shallowCompare = require('react-addons-shallow-compare');
-var styleEqual = require('style-equal');
+const shallowCompare = require('react-addons-shallow-compare'),
+      styleEqual = require('style-equal');
 
-var TRACK_SIZE = 4;
-var THUMB_SIZE = 20;
+const TRACK_SIZE = 4,
+      THUMB_SIZE = 20;
```

Comment:
nit: no need to change those, I prefer the way it was previously. But we can keep const.